scripts/sanitize.sh

```diff
@@ -48,7 +48,7 @@ if [[ $(uname -s) == 'Darwin' ]]; then
     #
     # See https://github.com/mapbox/node-cpp-skel/issues/122
     DYLD_INSERT_LIBRARIES=${MASON_LLVM_RT_PRELOAD} \
-      node node_modules/.bin/$(node -e "console.log(require('./package.json').scripts.test)") test/*test.js
+      node node_modules/.bin/tape test/*test.js
```

Comment:
This change was needed to avoid pulling in `nyc` here, which would break the ASAN run since the `DYLD_INSERT_LIBRARIES` will not be inherited by subprocesses (`tape` is a subprocess of `nyc`).